src/main/java/com/team2502/robot2020/command/autonomous/groups/AutonomousCommandGroupFactory.java

```diff
@@ -0,0 +1,172 @@
+package com.team2502.robot2020.command.autonomous.groups;
+
+import com.team2502.robot2020.Constants;
+import com.team2502.robot2020.command.*;
+import com.team2502.robot2020.command.autonomous.ingredients.*;
+import com.team2502.robot2020.subsystem.*;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class AutonomousCommandGroupFactory {
```

Comment:
one of these days we need to find a better way to make auto commands than this mess

Reply:
concur